kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -522,29 +558,6 @@ define([
             }
         };
 
-    const JobShortcutsMixin = (Base) =>
```

Comment:
removed these as they weren't all that useful (and they didn't have any tests!)

Reply:
😱